src/main/scala/com/gu/fastly/api/FastlyApiClient.scala

```diff
@@ -223,9 +222,9 @@ case class FastlyApiClient(apiKey: String, serviceId: String, config: Option[Asy
     AsyncHttpExecutor.execute(apiUrl, headers = commonHeaders)
   }
 
-  private def statsParams(from: DateTime, to: DateTime, by: By.Value, region: Region.Value = Region.all): Map[String, String] = {
-    def millis(date: DateTime): String = (date.getMillis / 1000).toString
-    Map[String, String]("from" -> millis(from), "to" -> millis(to), "by" -> by.toString, "region" -> region.toString)
+  private def statsParams(from: Instant, to: Instant, by: By.Value, region: Region.Value = Region.all): Map[String, String] = {
+    def seconds(date: Instant): String = date.getEpochSecond.toString
```

Comment:
It looks like this method was actually getting the seconds because it was dividing the milliseconds by 1000, so we've renamed it.

Reply:
The [Fastly API docs](https://www.fastly.com/documentation/reference/api/metrics-stats/stats/) are vague about what units `from` and `to` are actually in - they just say "Epoch timestamp" - but looking at the numbers given in the example section we see values like `1608560817` - which is far too small to be a recent milliseconds value!

```
curl -i  "https://api.fastly.com/service/SU1Z0isxPaozGVKXdv0eY/stats/summary?end_time=**1608560817**&month=05&start_time=**1608560817**&year=2020"
```

<img width="1160" height="608" alt="Image" src="https://github.com/user-attachments/assets/a986d166-cd69-4319-a3cf-e0d23278ccb9" />